fv3core/.jenkins/jenkins.sh

```diff
@@ -201,9 +203,22 @@ if [ ${python_env} == "virtualenv" ]; then
     fi
     export FV3_PATH="${JENKINS_DIR}/../"
     export TEST_DATA_RUN_LOC=${TEST_DATA_HOST}
+    if [ "$orchestration"  == "dace" ] ; then
```

Comment:
Especially if this logic has to be copy-pasted into multiple bash scripts, I really think we should include dace as part of "the pace environment" always.

Reply:
The problem here is not installing dace, the naming is confusing. What triggers the install of dace is through backend, specifically `gtc:dace` or `gtc:dace_gpu`. However, in `gtc:dace`, there're two modes: one without and one with dace orchestration. Basically if you want to have orchestrated `gtc:dace`, you need to set those environment flags.

Reply:
Never mind, I understand now. Including dace as part of install requirements.